knowage-vue/src/modules/documentExecution/olap/olapSidebar/OlapSidebar.vue

```diff
@@ -39,34 +33,34 @@
                 </div>
 
                 <div class="p-my-3">
-                    <label class="kn-material-input-label">{{ $t('documentExecution.olap.sidebar.tableFunctions') }}</label>
+                    <label v-if="tableFunctionsVisible" class="kn-material-input-label">{{ $t('documentExecution.olap.sidebar.tableFunctions') }}</label>
                     <div class="p-grid p-mt-1">
                         <div class="p-col-4">
                             <Button
                                 icon="far fa-caret-square-up"
                                 class="p-button-plain kn-button--secondary"
                                 :class="{ 'olap-sidebar-button-active': showParentMembers }"
                                 v-tooltip.top="$t('documentExecution.olap.sidebar.showParentMembers')"
-                                :disabled="!isButtonVisible('BUTTON_FATHER_MEMBERS')"
+                                v-if="isButtonVisible('BUTTON_FATHER_MEMBERS')"
                                 @click="onShowParentMemberClick"
                             />
                         </div>
                         <!-- <div class="p-col-4">
-                            <Button icon="fas fa-calculator" class="p-button-plain kn-button--secondary" v-tooltip.top="$t('documentExecution.olap.sidebar.calculatedFieldWizard')" :disabled="!isButtonVisible('BUTTON_CC')" />
+                            <Button icon="fas fa-calculator" class="p-button-plain kn-button--secondary" v-tooltip.top="$t('documentExecution.olap.sidebar.calculatedFieldWizard')" v-if="isButtonVisible('BUTTON_CC')" />
                         </div> -->
```

Comment:
Is this one for the CC? In case I will leave it

Reply:
Just leave it as is, it will never be shown because its filtered out of the request.